jetpack-io/devbox
# Auto-generate .envrc at devbox init

## Summary
Title says it
This is step 2 out of 4 for simplifying direnv integration with devbox

## How was it tested?
- compile
- run `devbox init`
- confirm a `.envrc` file is created
